templates/registration/register.html

```diff
@@ -7,15 +7,16 @@ <h2 class="extra-padding">Register</h2>
         <hr>
     </div>
     <div class="center">
-        <a href="{% url 'social:begin' backend='facebook-register' %}"><img class="oauth-icon" src="{% static "img/facebook-signin.png" %}" alt="Register with Facebook" /></a><br/>
-        <a href="{% url 'social:begin' backend='google-oauth2-register' %}"><img class="oauth-icon" src="{% static "img/google-signin.png" %}" alt="Register with Google" /></a>
+        <p>You can sign up with one click using facebook or google</p>
+        <a href="{% url 'social:begin' backend='facebook-register' %}"><img class="oauth-icon" src="{% static "img/facebook-signup.png" %}" alt="Register with Facebook" /></a><br/>
+        <a href="{% url 'social:begin' backend='google-oauth2-register' %}"><img class="oauth-icon" src="{% static "img/google-signup.png" %}" alt="Register with Google" /></a>
     </div>
     <div class="form-input">
+        <p>or create an account</p>
```

Comment:
-- not sure if you saw my comment above -- wasn't sure how to respond to your post about new new google logo so I just edited your comment ha